src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -27,22 +27,16 @@
 		:class="{deleted: !!deleteTimeout, disabled: !calendar.enabled, 'open-sharing': shareMenuOpen}"
 		@click.prevent.stop="toggleEnabled">
 		<template slot="icon">
-			<Actions>
-				<ActionButton @click.prevent.stop="toggleEnabled">
```

Comment:
There is now no way to toggleEnabled anymore. It is missing on the Icons.

Reply:
Ah, nevermind, you are right. I missed the @click handler on the `AppNavigationItem`. Sorry.